lambdas/access_counts/index.py

```diff
@@ -27,6 +27,9 @@
 # Timestamp for the dir above.
 LAST_UPDATE_KEY = f'{OBJECT_ACCESS_LOG_DIR}.last_updated_ts.txt'
 
+# Athena does not allow us to write more than 100 partitions at once.
+MAX_OPEN_PARTITIONS = 100
```

Comment:
Not a requirement for this PR but we should probably use [`boto3` service quotas](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/service-quotas.html#ServiceQuotas.Client.get_service_quota) instead of hardcoding.